ice.c

```diff
@@ -3404,13 +3404,40 @@ static gboolean janus_ice_outgoing_rtcp_handle(gpointer user_data) {
 		/* Free and reset stats list */
 		g_slist_free(stream->transport_wide_received_seq_nums);
 		stream->transport_wide_received_seq_nums = NULL;
-		/* Get feedback packet count and increase it for next one */
-		guint8 feedback_packet_count = stream->transport_wide_cc_feedback_count++;
-		/* Create rtcp packet */
-		int len = janus_rtcp_transport_wide_cc_feedback(rtcpbuf, size,
-			stream->video_ssrc, stream->video_ssrc_peer[0], feedback_packet_count, packets);
-		/* Enqueue it, we'll send it later */
-		janus_ice_relay_rtcp_internal(handle, 1, rtcpbuf, len, FALSE);
+
+		guint packets_len = 0;
+		while ((packets_len = g_queue_get_length(packets)) > 0) {
+			GQueue *packets_to_process;
+			// if packets_len > 400, transport_wide_cc packet could be overflow the buffer
```

Comment:
Please use `/*  */` for comments and not `//` (here and in a few of the following lines).